src/request.js

```diff
@@ -212,6 +212,7 @@ function connect(options, onSuccess, onError) {
 
 	options.backend({
 		url: options.url,
+		backendOptions: options.backendOptions,
```

Comment:
These options will pass through all the way to where we make a request into sockjs.